pkg/service/webhook/model.go

```diff
@@ -1,45 +1,57 @@
 package webhook
 
+import (
+	"fmt"
+	"net/url"
+)
+
 // In the context of webhooks, it's common to use noun.verb notation to describe events,
 // such as "credential.create" or "schema.delete".
 type Noun string
 type Verb string
```

Comment:
there's a risk here that noun and verb have whitespace or paths

meaning we need to make sure they're URL-safe

Reply:
not sure what you mean by 'URL' safe, these are strings not urls defined by us:
nouns:
credential
schema
did


verbs:
create
delete


they would run getsupporrtednouns and getsupportedverbs, find the strings they want that makes a valid noun.verb pairing (otherwise we wont accept and return error)

Reply:
ah my misunderstanding